src/services/exit_order_iterator.py

```diff
@@ -253,8 +229,7 @@ def _eject_validator(self, gid: NodeOperatorGlobalIndex) -> LidoValidator:
 
     def _no_predicate(self, node_operator: NodeOperatorStats) -> tuple:
         return (
-            node_operator.delayed_validators,
-            - self._no_force_predicate(node_operator),
+            self._no_force_predicate(node_operator),
```

Comment:
Bug over here
```suggestion
            - self._no_force_predicate(node_operator),
```
Pls, write test that sorting predicates correctly sorted (desc in this case)